backend/cmd/api/main.go

```diff
@@ -49,7 +53,55 @@ func main() {
 
 	authorizer := authorizer.NewAuthorizer(repo)
 	eventBroker := events.NewBroker()
-	scoreKeeper := scores.NewScoreKeeper()
+	scoreKeeper := scores.NewScoreKeeper(eventBroker)
+
+	go scoreKeeper.Run(ctx)
+
+	engine := scores.NewScoreEngine(1, eventBroker, &scores.HardestProblems{Number: 5}, scores.NewBasicRanker(3))
```

Comment:
Move this to a method which name makes it clear that this is temporary code for testing.